examples/czmq/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required (VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("${CMAKE_CURRENT_SOURCE_DIR}/../common.cmake")
+
+project(download-czmq)
+
+set(THREADS_PREFER_PTHREAD_FLAG ON)
```

Comment:
There is no `pthread` in user's code so it looks suspicious. Have you experienced any problems without this line?